helga_jeopardy.py

```diff
@@ -267,6 +361,170 @@ def scores(client, channel, nick, alltime=False):
+            return
+
+    game = {
+    'channel': channel,
+    'game_active': True,
+    'game_started': False,
+    'game_host': nick,
+    }
+    for category_num in range(1, 7):
+        category_name = 'cat{}'.format(category_num)
+        category = cat_dict[category_name].json()
+        game[category_name] = {'category': category['title']}
+        set_value = 200
+        for i in range(5):
+            clue_name = 'clue{}'.format(i + 1)
+            question = category['clues'][i]
+            game_question = {
+                'question': question['question'],
+                'answer': question['answer'],
+                'value': set_value,
+                'id': question['id'],
+                'active': True,
+            }
+            game[category_name][clue_name] = game_question
+            set_value += 200
+
+    game_id = db.jeopardy.insert(game)
+    client.msg(channel, "New game created. to join: !j game join")
+    return
+
+def start_new_game(nick, new_game, client, channel, mongo_db=db.jeopardy):
+    current_players = []
+    for i in new_game["players"]:
+        current_players.append(i)
+
+    if nick not in current_players:
+        client.msg(channel, "you have not joined the game lobby. try: !j game join")
+        return
+
+    for player in current_players:
+        mongo_db.update({
+            'game_active': True,
+            'game_started': False,
+        }, {
+            '$set': {
+                player: 0,
+           }
+        })
+    client.msg(channel, "Game started with the following players: {}".format(current_players))
+    client.msg(channel, "Here are today's categories:")
+    show_board(client, channel, mongo_db=db.jeopardy)
+    control = random.choice(current_players)
+    client.msg(channel, "By random choice, {} has control of the board".format(control))
+    client.msg(channel, "To select a question, use the following format: !j cat3 200")
+    mongo_db.update({
+        'game_active': True,
+        'game_started': False,
+    }, {
+        '$set': {
+            'game_started': True,
+            'control': control,
+       }
+    })
+    reactor.callLater(1800, end_game, client, channel)
+    return
+
+def show_board(client, channel, mongo_db=db.jeopardy):
+    current_game = mongo_db.find_one({
+        'channel': channel,
+        'game_active': True,
+    })
+
+    for key, value in current_game.items():
+        if key.startswith('cat'):
+            values = [str(value['clue{}'.format(i)]['value']) for i in range(1, 6) if value['clue{}'.format(i)]['active']]
+            cat_names = value["category"]
+            score_response =' '.join([key,  cat_names, ' '.join(sorted(values, key=int))])
+            client.msg(channel, score_response)
+
+    players = current_game['players']
+    client.msg(channel, "Here are the current scores:")
+    for i in players:
+        client.msg(channel, "{} : {}".format(i, current_game[i]))
```

Comment:
If you make the change I suggested about making a dict for each player, this could be:

```suggestion
    client.msg(channel, 'Here are the current scores:')
    for player in current_game['players']:
        client.msg(channel, '{}: ${}'.format(player['nick'], player['score']))
```